### PR TITLE
use session service to check if someone is staff

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

**Comment by keirog** [comment]
:+1:

File: app.js [hunk 1]
```diff
@@ -14,14 +15,22 @@ app.get('/__gtg', function(req, res) {
...
+
 app.get('/survey', function(req, res) {
 	res.set({ 'Cache-Control': 'max-age=0, private, no-cache' });
-
-	if (req.headers['x-ft-staff']) {
-		res.redirect('https://www.ftfeedbackforum.com/R.aspx?a=963');
-	} else {
-		res.redirect('https://www.ftfeedbackforum.com/R.aspx?a=1004');
-	}
+	fetch('https://session-next.ft.com').then(fetchres.json).then(function(json){
```
Comment (matthew-andrews) [inline app.js]: this is server side logic…… how does it know who the user is?

Comment (keirog) [inline app.js]: good point… https://github.com/Financial-Times/next-opt-in-out/blob/master/app.js#L125

Comment (aintgoin2goa) [inline app.js]: ah, yeah - forgot that bit

Comment (aintgoin2goa) [inline app.js]: https://github.com/Financial-Times/next-redirector/pull/41

**Pull request merged**